### ISSUE TITLE
Add homestead setup to README

### ISSUE THREAD

**Issue by Cyruxx** [laravel/spark#22]
Please provide a `homestead` section to the README that describes how to setup this on a homestead machine. Steps:

* Create new project `laravel new ...`
* Go to project `cd project/`
* Get laravel/homestead `composer require laravel/homestead`
* Make homestead `./vendor/laravel/homestead/homestead make`
* Modify Homestead.yaml to fit your needs, update /etc/hosts
* Run `vagrant up`
* Run `vagrant ssh`
* Run `cd project/`
* Install spark: `spark install`
* After installation process go back to your local machine project directory (e.g. in a new terminal tab)
* Run `npm install`
* Run `gulp`
* Go back to the vagrant machine terminal
* Run `php artisan vendor:publish`
* Run `php artisan migrate`

Done

Any suggestions?

**Comment by kylestev** [comment]
Running `spark install` prompts you if you want to run gulp and npm during the install process. Same with migrating your database.

**Comment by Cyruxx** [comment]
@kylestev You're right. Unfortunately you have to run the migrate command within the vagrant machine.

**Comment by phroggyy** [comment]
@Cyruxx not quite true. If you'd open up MySQL to external connections and then change your .env from `localhost` to the IP address, you could do it locally as well.

**Comment by Cyruxx** [comment]
@phroggyy That's a special case, because of the default .env file copied from the .env.example file contains localhost as host. Also, you had to open up MySQL to external connections as you said that'd be an extra step too.

But it's an option I guess.

**Comment by kylestev** [comment]
Homestead port forwards 3306 on the VM to port 33060 on your host machine. Simply change `DB_HOST=localhost:33060` and you can run commands from your host machine. Your VM will still be able to connect to the database using that host config.

**Comment by phroggyy** [comment]
@kylestev is MySQL on homestead open to external connections by default?

**Comment by kylestev** [comment]
It's open to your host machine (the machine running the VM/Homestead). You can find more information on the ports Homestead forwards for you out-of-the-box here: http://laravel.com/docs/5.1/homestead#ports

**Comment by mikebronner** [comment]
This has been my solution: https://dev-environment.forlaravel.com/homestead-commands#and-the-icing
